### DISCUSSION TITLE
Automation account module don't pass all parameters for runbooks

### DISCUSSION THREAD

**Discussion by dibcar** [Azure/ResourceModules#3282] [Q&A] [answered]
Hi, 
Is there a reason why the automation account module don't pass every parameter to the runbook sub module ? The parameters scriptStorageAccountId and sasTokenValidityLenght are not passed to the runbook module.

Thanks,

**Comment by eriqua** [comment, marked as answer]
Hi @dibcar, thanks for opening this discussion and sorry for the late reply. Discussions are currently not monitored promptly and are planned to be disabled in favor of the new issue template [Questions/Feedback].

What you are raising looks like a missing functionality and I'm converting this discussion to a Feature request issue in order for the team to be able to triage it.

Please refer to issue #3291  going forward.
